README.md

```diff
@@ -219,39 +222,45 @@ _Image 3-6，Classification Settings_
 
-#### 3.2.3 Setting advanced Setting
+#### 3.2.3 Rework times Setting
+![Image 3-7](https://cdn.jsdelivr.net/gh/au-heartbeat/data-hosting@main/rework-setting-image/rework-times-settings.png)\
+_Image 3-7，Rework times Settings_
 
-![Image 3-7](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/advance-settings.png)\
-_Image 3-7，advanced Settings_
+In Rework times settings, it contains Rework to which state Input and Exclude which states(optional) Input. The options in the Rework to which state Input are all from Board mappings, the options are ordered, and when an option is selected, the rework information of the option and all subsequent options will be counted in the report page and export file. The Exclude which states(optional) Input can help you exclude certain subsequent options (image 3-7).
+
+#### 3.2.4 Setting advanced Setting
+
+![Image 3-8](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/advance-settings.png)\
+_Image 3-8，advanced Settings_
 
 In advanced settings, it contains story points Input and Flagged Input. Users can input story points and Flagged custom-field on their own when the jira board has permission restriction . And according to these input, in the export page, user can get correct story points and block days
 
 how to find the story points and Flagged custom-field?
 
-![Image 3-8](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/devtool-network.png)\
-_Image 3-8，devTool-network-part_
+![Image 3-9](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/devtool-network.png)\
+_Image 3-9，devTool-network-part_
 
-![Image 3-9](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/card-history.png)\
-_Image 3-9，card-history_
+![Image 3-10](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/card-history.png)\
+_Image 3-10，card-history_
 
-![Image 3-10](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/find-custom-field-api.png)\
-_Image 3-10，find-custom-field-api_
+![Image 3-11](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/find-custom-field-api.png)\
+_Image 3-11，find-custom-field-api_
 
-![Image 3-11](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/story-point-custom-field.png)\
-_Image 3-11，story-point-custom-field_
+![Image 3-12](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/story-point-custom-field.png)\
+_Image 3-12，story-point-custom-field_
 
-![Image 3-12](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/flagged-custom-field.png)\
-_Image 3-12，flagged-custom-field_
+![Image 3-13](https://jsd.cdn.zzko.cn/gh/au-heartbeat/data-hosting@main/advanced-setting-image/flagged-custom-field.png)\
+_Image 3-13，flagged-custom-field_
```

Comment:
use `https://cdn.jsdelivr.net` this cdn host, not use others, 
just keep the same, we can change it quickly if there is any cdn block